src/reducer/reducer.js

```diff
@@ -0,0 +1,83 @@
+import {ADD_TODO, REMOVE_TODO} from '../actions/action.js'
```

Comment:
No I wrote it out as todo because I found it easier to think of it in terms of the todo app. I wrote todo_item for all the different items that were clicked on.